src/Libraries/Microsoft.AspNetCore.Diagnostics.Middleware/Microsoft.AspNetCore.Diagnostics.Middleware.csproj

```diff
@@ -22,7 +23,7 @@
 
   <PropertyGroup>
     <Stage>normal</Stage>
-    <MinCodeCoverage>100</MinCodeCoverage>
+    <MinCodeCoverage>99</MinCodeCoverage>
```

Comment:
If coverage fell, it might indicate that the code generator is forgetting to annotate something with [GeneratedCode], which was the case with the logging generator.

Reply:
If this is the case, we shoud:

* Fix the code generator.
* Add an exclusion to eng/Common.runsettings to exclude the generated files.

Reply:
The config binder source generator does add `[GeneratedCode]` attributes. I'm reverting this change to see what the problem was - this change was ported from the 9.0 branch in #4625.

Reply:
I figured out why coverage dropped, see https://github.com/dotnet/extensions/pull/4871#issuecomment-1888094778. I will keep MinCodeCoverage=100 here.